lib/kernel/src/logger_formatter.erl

```diff
@@ -531,6 +531,9 @@ check_template([Str|T]) when is_list(Str) ->
         true -> check_template(T);
         false -> error
     end;
+check_template([Bin|T]) when is_binary(Bin) ->
+    Str = binary:bin_to_list(Bin),
```

Comment:
this wouldn't be unicode-safe compared to `unicode:characters_to_list/2-3`.

Reply:
Fixed.